### DISCUSSION TITLE
less strict file filter

### DISCUSSION THREAD

**Discussion by msconfig87** [warrenbuckley/Compact-Log-Format-Viewer#341] [Ideas]
Hi, I like your log viewer, it displays the logs very clean. 
Here is a suggestion: During initial file open dialog you restrict files to ".txt", ".json" and ".clef".
I have to rename all my logs to ".txt" first to open them, which can be time consuming.
Can you add ".log" as well? Or maybe even better a "*.*" option, because some tools just use their own ending even if the format is correct inside.
cheers

**Comment by warrenbuckley** [comment]
This has now been resolved :)
